templates/components/cv_overview_table.md

```diff
@@ -0,0 +1,5 @@
+| Entry | Substrate | Electrolyte | year | Reference |
```

Comment:
@DunklesArchipel I changed this table to be more reusable. It now expects a database as input and renders a table, i.e., there is no reference to materials anymore.